neo/Network/P2P/Payloads/CosignerUsage.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Neo.IO;
+using Neo.IO.Json;
+using System;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class CosignerUsage : ISerializable
+    {
+        public WitnessScope Scope;
+        public UInt160 ScriptHash;
+        public int Size =>
+          Scope.Size +
+          ScriptHash.Size;
+
+        void ISerializable.Deserialize(BinaryReader reader)
+        {
+            Scope = reader.ReadSerializable<WitnessScope>();
+            ScriptHash = reader.ReadSerializable<UInt160>();
+        }
+
+        void ISerializable.Serialize(BinaryWriter writer)
+        {
+            writer.Write(Scope);
+            if(ScriptHash != null)
```

Comment:
This must be filled right?

Reply:
I think so... it was giving me many nullpointer exceptions, so I put this if. It was caused by tests not filling this,now we can fix.

Reply:
Fixed!